clients/imodels-client-management/src/operations/SharedFunctions.ts

```diff
@@ -32,3 +32,8 @@ export function assertStringHeaderValue(headerName: string, headerValue: unknown
   if (!isString)
     throw new Error(`Assertion failed: header's ${headerName} value is not a string.`);
 }
+
+export function assertLink(link: Link | null | undefined): asserts link is Link {
```

Comment:
I moved the assert from authoring to management, but I'm not sure about it.. it's kind of a breaking change

Reply:
I don't think it was ever exported from the @itwin/imodels-client-authoring package
https://github.com/iTwin/imodels-clients/blob/main/clients/imodels-client-authoring/src/index.ts
it should not be an issue